### DISCUSSION TITLE
License purchase Issue

### DISCUSSION THREAD

**Discussion by guruprasadk-reconn** [serenity-is/Serenity#7000] [Help]
As a startup, we were interested in purchasing a Business License and approached the Serenity Sales Team to inquire about discounted pricing. They shared a time-limited link for the purchase. Unfortunately, we faced challenges in making the payment within the stipulated timeframe because the available payment methods did not align with our preference, as we intended to use Net Banking. Despite our repeated attempts to reach out to the team via email, we did not receive any response.

In light of the absence of communication from the sales team, I am expressing our concern here. Please suggest how to go about it.

Guruprasad K,
India

**Comment by volkanceylan** [comment]
As the sales team replied to you before clearly, it was a "limited time" offer which expired long ago. We are not going to offer any discounts again.

**Comment by guruprasadk-reconn** [comment]
We request you to reconsider for one more time.
